docs/src/content/docs/it/reference/frontmatter.md

```diff
@@ -117,3 +133,149 @@ interface HeroConfig {
   }>;
 }
 ```
+
+### `banner`
+
+**type:** `{ content: string }`
+
+Visualizza un banner di annuncio nella parte superiore di questa pagina.
+
+Il valore `content` può includere HTML per collegamenti o altri contenuti.
+Ad esempio, questa pagina visualizza un banner che include un collegamento a `example.com`.
+
+```md
+---
+title: Pagina con un banner
+banner:
+  content: |
+    Abbiamo appena lanciato qualcosa di interessante!
+    <a href="https://example.com">Dai un'occhiata</a>
+---
+```
+
+### `lastUpdated`
+
+**type:** `Date | boolean`
+
+Sostituisce l'[opzione globale `lastUpdated`](/reference/configuration/#lastupdated). Se viene specificata una data, deve essere un [timestamp YAML](https://yaml.org/type/timestamp.html) valido e sovrascriverà la data archiviata nella cronologia Git per questa pagina.
```

Comment:
Qua possiamo usare un collegamento alla pagina `/it/`:

```suggestion
Sostituisce l'[opzione globale `lastUpdated`](/it/reference/configuration/#lastupdated). Se viene specificata una data, deve essere un [timestamp YAML](https://yaml.org/type/timestamp.html) valido e sovrascriverà la data archiviata nella cronologia Git per questa pagina.
```

Reply:
You are right! I did it in the normal Astro docs which had the `/en/`, totally forgot it here as it was root (why the difference, though?). It's gonna be same in most PRs, will update them all later!

Reply:
The ability to use a root locale is something we got a lot in relation to i18n so it seemed like we needed to support it for a general tool like Starlight even though it does add some code complexity. We use it for the Starlight docs to dog food it and catch bugs.

Reply:
Oh, gotcha! Makes sense then.

It's annoying for the various imports as they change the depth when you move them over to any language ahah I'm tempted to make a PR to use aliases.